edit.php

```diff
@@ -14,10 +14,10 @@
     <title>Edit Pokemon</title>
 </head>
 <body>
-<h1>Edit Pokemon</h1>
+<h1>Edit <?=($userPokemon['hasNickname']==1) ? $userPokemon['nickname'] . " the " : ""; echo $userPokemon['species'];?></h1>
 <?php echo displayPokemon($userPokemon,0); ?>
-<form>
-    <label>Give <?php echo $userPokemon['nickname'] ?? $userPokemon['species'];?> a new nickname?
+<form method="post" action="editNickname.php">
+    <label>Give <?php echo $userPokemon['nickname'] ?? $userPokemon['species'];?> a new nickname? (Leave blank to clear nickname. Max chars: 12)
```

Comment:
Maybe you could have a remove nickname button instead?

Reply:
Have considered it, but prefer this method for now. Might add later though!